docs/tooling/fcl-js/api.md

```diff
@@ -285,47 +284,88 @@ addStuff().then((d) => console.log(d)); // 13 (5 + 7 + 1)
 | `flow.network` **(recommended)**                      | `testnet`                                            | Used in conjunction with stored interactions and provides FCLCryptoContract address for `testnet` and `mainnet`. Possible values: `local`, `canarynet`, `testnet`, `mainnet`.                                                                            |
+## Using Contracts in Scripts and Transactions
 
-### Address replacement in scripts and transactions
+### Address Replacement
 
-Configuration keys that start with `0x` will be used to find-and-replace their values in Cadence scripts and transactions input to FCL. Typically this is used to represent account addresses. Account addresses for the same contract will be different depending on the Flow network you're interacting with (eg. Testnet, Mainnet).
-This allows you to write your script or transaction once and not have to update code when you point your application at a different Flow network.
+Configuration keys that start with `0x` will be replaced in FCL scripts and transactions, this allows you to write your script or transaction Cadence code once and not have to change it when you point your application at a difference instance of the Flow Blockchain.
 
 ```javascript
-import * as fcl from "@onflow/fcl";
+import * as fcl from "@onflow/fcl"
 
-fcl
-  .config()
-  .put("accessNode.api", "https://rest-testnet.onflow.org")
-  .put("0xFlowToken", "0x7e60df042a9c0868");
+fcl.config()
+  .put("0xFungibleToken", "0xf233dcee88fe0abe")
 
-async function myScript() {
-  return fcl.query({
-    cadence: `
-        import FlowToken from 0xFlowToken // will be replaced with 0xf233dcee88fe0abe because of the configuration
+async function myScript () {
+  return fcl.send([
+    fcl.script`
+      import FungibleToken from 0xFungibleToken // will be replaced with 0xf233dcee88fe0abe because of the configuration
 
-        pub fun main(): UFix64 { 
-          return FlowToken.totalSupply  // arbitrary script that can access FlowToken interface
-        }
-      `,
-  });
+      pub fun main() { /* Rest of the script goes here */ }
+    `
+  ]).then(fcl.decode)
+}
+
+async function myTransaction () {
+  return fcl.send([
+    fcl.transaction`
+      import FungibleToken from 0xFungibleToken // will be replaced with 0xf233dcee88fe0abe because of the configuration
+
+      transaction { /* Rest of the transaction goes here */ }
+    `
+  ]).then(fcl.decode)
 }
 ```
 
-### Example
+#### Example
 
 ```javascript
-import * as fcl from "@onflow/fcl";
+import * as fcl from "@onflow/fcl"
 
-fcl
-  .config()
+fcl.config()
   .put("flow.network", "testnet")
   .put("accessNode.api", "https://rest-testnet.onflow.org")
-  .put("discovery.wallet", "https://fcl-discovery.onflow.org/testnet/authn")
+  .put("discovery.wallet", "http://localhost:3000/fcl/authn")
```

Comment:
Should we not use the deployed FCL Discovery for testnet?

Reply:
good catch